pkg/apis/apps/v1alpha1/servicebindingrequest_types.go

```diff
@@ -66,7 +66,8 @@ type BackingServiceSelector struct {
 	metav1.GroupVersionKind `json:",inline"`
 	ResourceRef             string `json:"resourceRef"`
 	// +optional
-	Namespace *string `json:"namespace,omitempty"`
+	Namespace    *string `json:"namespace,omitempty"`
+	EnvVarPrefix string  `json:"envVarPrefix,omitempty"`
```

Comment:
Maybe use `EnvVarPrefix *string` so that we can distinguish between 
* a skipped field, and
* an empty string
in our code.

Asking for this because:
* an empty string explicitly implies that the environment variable need to looks like `HOST`
* a skipped field implicitly implies that the environment variable needs to look like `KIND_HOST`